modules/apps/address/address-test/src/testIntegration/java/com/liferay/address/service/test/CountryLocalServiceTest.java

```diff
@@ -140,6 +147,76 @@ public void testDeleteCountry() throws Exception {
 					QueryUtil.ALL_POS)));
 	}
 
+	@Test
+	public void testSearchCountries() throws Exception {
+		String keywords1 = RandomTestUtil.randomString();
+		String keywords2 = RandomTestUtil.randomString();
+		String keywords3 = keywords1 + RandomTestUtil.randomString();
+
+		Country country1 = _addCountry("a1", "a11", true, keywords1);
+		Country country2 = _addCountry("a2", "a22", true, keywords2);
+		Country country3 = _addCountry("a3", "a33", true, keywords3);
+
+		List<Country> expectedCountries = Arrays.asList(country1, country3);
```

Comment:
The country1 and country3 can be omitted since they are not being used. You can simplify this by doing

```
Arrays.asList(_addCountry(...), _addCountry(...));
```